cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java

```diff
@@ -166,6 +166,8 @@ public static HiveSupport checkHiveSupport(@Nullable ClassLoader hiveClassLoader
       return HiveSupport.HIVE_1_1;
     }  else if (hiveVersion.startsWith(("1.2"))) {
       return HiveSupport.HIVE_1_2;
+    } else if (hiveVersion.startsWith(("2.1"))) {
```

Comment:
HDP 2.6 uses Hive 2.1 and same with EMR 5.2. I don't think there is a distribution with Hive 2.0. We can add that later if required.